mbed-hal/spi_api.h

```diff
@@ -112,54 +166,45 @@ void spi_frequency(spi_t *obj, int hz);
  *
  * @param[in] obj   The SPI peripheral to use for sending
  * @param[in] value The value to send
- * @return Returns the value received during send
+ * @return returns the value received during send
  */
 int  spi_master_write(spi_t *obj, int value);
```

Comment:
Yup, but I was told not to break everything right away.
We can remove them later on.